packages/core/stories/layout/grid-layout.doc.stories.mdx

```diff
@@ -25,76 +26,87 @@ For more complex layouts, it supports spanning across columns and rows.
   <Story id="core-layout-gridlayout--default-grid-layout" />
 </Canvas>
 
-## How you can use Grid layout
+## Use Grid layout when...
+
+Your content is best displayed using a configurable number of columns and rows.
 
 Grid layout can take any number of children, and will by default display them in a 12 columns grid (items after the 12th one will wrap).
-to change the number of columns displayed in the grid, you can reduce or increase the `columns` prop.
+In order to change the number of columns displayed in the grid, you can reduce or increase the `columns` prop.
 
 Grid layout is useful when you want to control the layout in both row and columns dimensions and it can achieve more complex forms when used in combination with `GridItems`.
-If you want to control the flow in a single dimension, we recommend you having a look at [Stack layout](?path=/docs/documentation-core-layout-stacklayout--page), [Flow layout](?path=/docs/documentation-core-layout-flowlayout--page) or [Flex layout](?path=/docs/documentation-core-layout-flexlayout--page).
 
-## API
+### Don't use Grid layout when...
 
-```
-import { GridLayout, GridItem } from "@jpmorganchase/uitk-core";
-```
+You want your layout to flow in a single dimension. In this case we recommend you having a look at [Stack layout](?path=/docs/documentation-core-layout-stacklayout--page), [Flow layout](?path=/docs/documentation-core-layout-flowlayout--page) or [Flex layout](?path=/docs/documentation-core-layout-flexlayout--page).
 
-## Props
+### Examples
 
-**ResponsiveProp** refers to a prop than take either multiple values (one per breakpoint) or a single value that would take effect across all different screen sizes.
```

Comment:
I think this is happening in more places, might be worth checking all the example's text